test/e2e/framework/pod/resource.go

```diff
@@ -193,7 +193,7 @@ func podRunningAndReady(c clientset.Interface, podName, namespace string) wait.C
 		}
 		switch pod.Status.Phase {
 		case v1.PodFailed, v1.PodSucceeded:
-			e2elog.Logf("The status of Pod %s is %s which is unexpected", podName, pod.Status.Phase)
```

Comment:
should this also be added at line 202?

Reply:
I think adding to line 202 may print more useless logs, only printing the details while pod becomes terminal status could be enough, right?